### PR TITLE
Classifier may be null, improved checking to avoid NPE.

### PR DESCRIPTION
**Pull request by nelsongraca**
Made what I believe to be a small improvement, hope it helps.

### CODE REVIEW

File: plugin/src/main/java/org/wildfly/swarm/plugin/maven/MultiStartMojo.java [hunk 1]
```diff
@@ -186,7 +186,10 @@ protected MavenProject findProject(String groupId, String artifactId) {
     protected Artifact findArtifact(String groupId, String artifactId, String classifier) {
         return this.project.getArtifacts()
                 .stream()
-                .filter(e -> (e.getGroupId().equals(groupId) && e.getArtifactId().equals(artifactId) && e.getClassifier().equals(classifier)))
+                .filter(e -> (e.getGroupId().equals(groupId) && e.getArtifactId().equals(artifactId)
+                        && ((e.getClassifier() != null e.getClassifier().equals(classifier)) || e.getClassifier() == null && classifier == null)
```
Comment (gastaldi) [inline plugin/src/main/java/org/wildfly/swarm/plugin/maven/MultiStartMojo.java]: This doesn't seem to compile

Comment (nelsongraca) [reply plugin/src/main/java/org/wildfly/swarm/plugin/maven/MultiStartMojo.java]: Will double-check.

Comment (nelsongraca) [reply plugin/src/main/java/org/wildfly/swarm/plugin/maven/MultiStartMojo.java]: Seems to be checkstyle, can you help with that?

**Comment by nelsongraca** [comment]
Now it compiles.

**Comment by nelsongraca** [comment]
Another fix for a possible NPE.

**Comment by kenfinnigan** [comment]
ok to test

**Comment by kenfinnigan** [comment]
Thanks for this fix!

In the future, would be appreciated if you could also create an issue in [JIRA](https://issues.jboss.org/browse/SWARM)

**Comment by nelsongraca** [comment]
Will do in the future.

**Pull request merged**